kubernetes-services/work-updater/README.md

```diff
@@ -0,0 +1,35 @@
+# Harmony Work Updater
+
+This folder contains the code and build scripts for the Harmony work updater,
+which is responsible for reading work-item updates from a queue and handling processing them.
+
+## Building the Work Updater
+
+1. Run `npm build` in this directory to build with the tag `latest`. To use a different tag,
+   run `VERSION=tag npm run build`.
+
+## Using the Updater Locally
+
+1. Set `USE_SERVICE_QUEUES=true` in your .env file for the harmony repository
```

Comment:
Isn't this the default? I don't think we need this step.

Reply:
Yeah, it is now. I'll remove this. Done.